docs/intro/overview.rst

```diff
@@ -12,6 +12,14 @@ Even though Scrapy was originally designed for `web scraping`_, it can also be
 used to extract data using APIs (such as `Amazon Associates Web Services`_) or
 as a general purpose web crawler.
 
+Why use Scrapy?
+===============
+
+Scrapy takes care of the dirty work so you can focus on data extraction instead of boilerplate code.
+
+Scrapy handles requests and provides :ref:`selectors <topics-selectors>` so you can easily extract data using CSS or XPath. Its pipeline architecture makes it simple to process, validate and output data.
```

Comment:
`handles requests` seems a bit of hand-waving, considering it's about an important part of Scrapy that is difficult to implement manually. :)

We could mention advantages of using Scrapy for crawling like:

1) you can _configure_ politeness (i.e., no need to worry about it while writing logic of links to follow)
2) it is efficient (callbacks force you to handle every response in a separate function, which enables better resource usage)
3) it is extensible (you can build and reuse middlewares for crawling logic too, e.g. gather stats, choose proxies, handle special headers, pre-process requests/responses in general)

+1 on adding `Why use Scrapy`, I only feel this is focusing too much on the data extraction parts (which isn't THE great thing about Scrapy).

@rdowinton What do you think?